lib/index.js

```diff
@@ -193,7 +193,17 @@ Nightwatch.cli = function(callback) {
       throw new Error('Supplied callback argument needs to be a function.');
     }
 
-    callback(argv);
+    if (argv.sequential){
+      // Enable running tests in sequential mode for supported and provided environments
+      let envs = argv.env && argv.env.split(',');
```

Comment:
also you need to check here if argv.env is a string. e.g. this will throw an error:
```
let argv = {}
let envs = argv.env && argv.env.split(',');
envs.reduce()
```